plugins/Geant4Output2EDM4hep_DRC.cpp

```diff
@@ -409,7 +409,11 @@ void Geant4Output2EDM4hep_DRC::saveParticles(Geant4ParticleMap* particles) {
       if (mcp.isCreatedInSimulation())
         mcp.setGeneratorStatus(0);
 
+#if EDM4HEP_BUILD_VERSION <= EDM4HEP_VERSION(0, 99, 2)
```

Comment:
we have the same problem as https://github.com/AIDASoft/DD4hep/pull/1488#issuecomment-3187458923 , no?